Experiments/Experiments/FeatureFlag.swift

```diff
@@ -82,6 +82,10 @@ public enum FeatureFlag: Int {
     ///
     case justInTimeMessagesOnDashboard
 
+    /// Adds the System Status Report to a support requests
```

Comment:
```suggestion
    /// Adds the System Status Report to support requests
```
Super minor non-blocker change